### ISSUE TITLE
after session.bye it is not possible to establish a new one

### ISSUE THREAD

**Issue by celevra** [onsip/SIP.js#199]
Hi,

if i call someone (no problem at all we can speak and hear both sides) and end the session with .bye it is not possible to call him again, i get a simple closing INVITE session.
this is my code

```
    sipconfig = {
        uri: 'meet-'+profile.id+'@lionkeen.net',
        wsServers: 'wss://sip.lionkeen.net:7443',
        stunServers: ["stun:stun.l.google.com:19302"],
        authorizationUser: 'meet-'+profile.id,
        password: profile.sippw
    };
    sip = new SIP.UA(sipconfig);

var sipoptions = {
    media: {
        constraints: {
            audio: true,
            video: false,
        },
        render: {
            remote: {
                audio: document.getElementById('audio-remote')
            }
        }
    }
};

function openCall() 
{
    session = sip.invite('conf-'+profile.conf+'_'+profile.pin, sipoptions);
    session.on('accepted', onAccepted);
    session.on('bye', onEnd);
}

function onAccepted ()
{
    inCall = true;
    console.log('call established');
}

function onEnd ()
{
    inCall = false;
    console.log('call ended');
}

$('.phone').on('click', function(){
    if (inCall)
    {
        session.bye();
    }
    else
    {
        openCall();
    }
})
```

this is the log:
```
starting call
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event progress
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event accepted
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event rejected
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event failed
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event cancel
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event connecting
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event terminated
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event dtmf
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event invite
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | skipping event cancel - Event exists
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event refer
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event bye
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event hold
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event unhold
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event muted
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.clientcontext | adding event unmuted
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event userMediaRequest
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event userMedia
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event userMediaFailed
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event userMediaRequest
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event userMedia
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event userMediaFailed
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event iceGathering
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event iceComplete
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event iceFailed
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event getDescription
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event setDescription
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event dataChannel
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | adding event addStream
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | new listener added to event userMediaRequest
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | new listener added to event userMedia
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | new listener added to event userMediaFailed
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | acquiring local media
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.inviteclientcontext | new listener added to event accepted
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.inviteclientcontext | new listener added to event bye
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event userMediaRequest
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event userMediaRequest
adapter.js:318 spec:   {"audio":true,"video":false}
adapter.js:321 chrome: {"audio":true,"video":false}
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event userMedia
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event userMedia
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | acquired local media stream
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.inviteclientcontext | emitting event connecting
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:3670469233 1 udp 2122260223 10.93.97.176 62787 typ host generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:2999745851 1 udp 2122194687 192.168.56.1 62788 typ host generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:2278329664 1 udp 2122129151 10.9.0.2 62789 typ host generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:3670469233 2 udp 2122260222 10.93.97.176 62790 typ host generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:2999745851 2 udp 2122194686 192.168.56.1 62791 typ host generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:2278329664 2 udp 2122129150 10.9.0.2 62792 typ host generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:2487555713 1 tcp 1518280447 10.93.97.176 0 typ host tcptype active generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:4233069003 1 tcp 1518214911 192.168.56.1 0 typ host tcptype active generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:3377426864 1 tcp 1518149375 10.9.0.2 0 typ host tcptype active generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:2487555713 2 tcp 1518280446 10.93.97.176 0 typ host tcptype active generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:4233069003 2 tcp 1518214910 192.168.56.1 0 typ host tcptype active generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:19 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE candidate received: candidate:3377426864 2 tcp 1518149374 10.9.0.2 0 typ host tcptype active generation 0
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:28 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceCandidate
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:28 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE Gathering Completed
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:28 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event iceComplete
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:28 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | emitting event getDescription
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:28 GMT+0200 (Central Europe Daylight Time) | sip.ua | emitting event newTransaction
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:28 GMT+0200 (Central Europe Daylight Time) | sip.transaction.ict | adding event stateChanged
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:28 GMT+0200 (Central Europe Daylight Time) | sip.transaction.ict | emitting event stateChanged
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.transaction.ict | emitting event stateChanged
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.inviteclientcontext | closing INVITE session egqbuv1i5ab37qhgva2g41a459bvru
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | closing PeerConnection
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.inviteclientcontext | emitting event failed
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.inviteclientcontext | emitting event rejected
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.transaction.ict | Timer D expired for INVITE client transaction z9hG4bK7735338
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.transaction.ict | emitting event stateChanged
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.ua | emitting event transactionDestroyed
sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.invitecontext.mediahandler | ICE connection state changed to "closed"
```

am i doing something wrong, is there anything else i can provide?

regards

Celevra

**Comment by wakamoleguy** [comment]
Hi Celevra,

Based on this line in the logs:

> sip-0.6.4.min.js:36 Tue Jul 21 2015 14:12:29 GMT+0200 (Central Europe Daylight Time) | sip.inviteclientcontext | emitting event rejected

...it looks like you are receiving a rejection response over the wire.  This could be for any number of reasons.  Turning on the `traceSip` option on your UA would provide more details as to what is happening.  Could you enable that option, and post the resulting logs (ideally from both the working and nonworking calls) in a [Gist](https://gist.github.com/)?

Thanks,

-Will

**Comment by celevra** [comment]
Hi,

thank you very much for your help, here is the gist from the beginning with the register, first invite, bye and second invite

https://gist.github.com/celevra/cd383e0c1c9593a0ec60

in the second invite is a "SIP/2.0 400 Bad Content-Type Header"
hope that helps

regards

philipp

**Comment by wakamoleguy** [comment]
It looks like you're getting caught by bug #143, and some double headers are being appended in the second INVITE request.  This was fixed and merged in version 0.7.0.  I believe that Issue thread had a workaround in it as well, but I would recommend updating to 0.7.0 since it has several other bug fixes as well.

Cheers,

-Will
